PackageExplorer/Converters/NumberToStringConverter.cs

```diff
@@ -13,6 +13,10 @@ public class NumberToStringConverter : IValueConverter
             {
                 return i.ToMetric(decimals: 1);
             }
+            if (value is long l)
+            {
+                return ((double)l).ToMetric(decimals: 1);
```

Comment:
Humanizer does not support `long.ToMetric` https://github.com/Humanizr/Humanizer/issues/1153